javascript/javascript2/week1/Js2Week1.js

```diff
@@ -0,0 +1,102 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+const lengthArr = [];
+let min;
+let shortestWord;
+
+function findShortestWord() {
+
+    for (let i = 0; i < danishWords.length; i++) {
+        lengthItem = danishWords[i].length;
+        lengthArr.push(lengthItem);
+        lengthArr.sort((a, b) => a - b)
```

Comment:
It is unnecessary to use an array as a counter, and then sorting it on each iteration. Since you only want the shortest word, just keep the length of the shortest word and log that.

Reply:
yes ..did it